demo/play-demo-for-recording.sh

```diff
@@ -0,0 +1,208 @@
+#!/bin/bash
+
+#asciinema rec -w 2
+
+exec() {
+  echo "\$ $@"|pv -qL 10 ; "$@" ;
+  }
+
+echo "   "
+echo "   "
+echo " Simplify the deployment of Spring Boot applications using Halkyon Component Operator on Kubernetes"|pv -qL 10
+echo " In this demo, we will :"|pv -qL 10
+echo "   --> Compose & link microservices"|pv -qL 10
+echo "   --> Deploy a capability such as a database and link it to a microservice consuming it"|pv -qL 10
```

Comment:
`--> Deploy a capability such as a database and link it to a microservice consuming it` -> `--> Deploy a capability such as a database and link it to backend microservice accesing it`